pkgs/development/node-packages/node-env.nix

```diff
@@ -496,7 +496,7 @@ let
     let
       extraArgs = removeAttrs args [ "name" "dependencies" "buildInputs" "dontStrip" "dontNpmInstall" "preRebuild" "unpackPhase" "buildPhase" "meta" ];
     in
-    stdenv.mkDerivation (finalAttrs: {
```

Comment:
@Scrumplex this reverted your https://github.com/NixOS/nixpkgs/commit/4549773edf119c40f3285fd46b662a5ceec2e456